Firmware/variants/1_75mm_MK3S-EINSy10a-E3Dv6full.h

```diff
@@ -267,7 +273,8 @@
 
 //new settings is possible for vsense = 1, running current value > 31 set vsense to zero and shift both currents by 1 bit right (Z axis only)
 #define TMC2130_CURRENTS_H {16, 20, 35, 30}  // default holding currents for all axes
-#define TMC2130_CURRENTS_R {16, 20, 35, 30}  // default running currents for all axes
+#define TMC2130_CURRENTS_FARM 36             // E 805 mA peak for ECool/farm mode
+#define TMC2130_CURRENTS_R {16, 20, 35, 30}  // default running currents for all axes, the last parameter is E 805 mA peak for ECool mode
```

Comment:
Comment after the comma `, the last parameter is E 805 mA peak for ECool mode`
is misleading now.